homeassistant/components/logger/const.py

```diff
@@ -13,15 +13,13 @@
 LOGSEVERITY_WARNING = "WARNING"
 LOGSEVERITY_ERROR = "ERROR"
 LOGSEVERITY_CRITICAL = "CRITICAL"
-LOGSEVERITY_WARN = "WARN"
 LOGSEVERITY_FATAL = "FATAL"
 
 LOGSEVERITY = {
     LOGSEVERITY_CRITICAL: logging.CRITICAL,
     LOGSEVERITY_FATAL: logging.FATAL,
     LOGSEVERITY_ERROR: logging.ERROR,
     LOGSEVERITY_WARNING: logging.WARNING,
-    LOGSEVERITY_WARN: logging.WARN,
```

Comment:
Removing this is a breaking change. I don't think we should do that.

We can replace the value with `logging.WARNING` though.

Reply:
Yes, you are right. There are no instances in this codebase, but there may be others who use it.